core/object/script_instance.cpp

```diff
@@ -32,6 +32,28 @@
 
 #include "core/object/script_language.h"
 
+int ScriptInstance::get_method_argument_count(const StringName &p_method, bool *r_is_valid) const {
```

Comment:
Simple delegation here, very much like how it's done in GDScript but without the dedicated features used there, allows for using this feature without extending `ScriptInstance`, though it might not catch all cases, and might not be as optimal